### PR TITLE
add LoadLedger to horizon package

### PR DESCRIPTION
**Pull request by jedmccaleb**

### CODE REVIEW

**Comment by jedmccaleb** [comment]
We should discus how we want to handle dealing with the json structs returned from horizon in these calls. I feel like we want to gussie up the results to be more user friendly.

File: horizon/main.go [hunk 1]
```diff
@@ -50,6 +53,33 @@ func (c *Client) LoadAccount(accountId string) (account Account, err error) {
...
+			return
+		}
+		var ledgerList LedgerList
+		err = decodeResponse(resp, &ledgerList)
+		if err != nil || len(ledgerList.Embedded.Records)<1 {
+			return
+		}
+		ledger=ledgerList.Embedded.Records[0]
+	}else {
+		resp, err = c.Client.Get(c.URL + "/ledgers/" + strconv.FormatUint(ledgerNum,10))
```
Comment (nullstyle) [inline horizon/main.go]: This can be simplified:

```go
resp, err = c.Client.Get(fmt.Sprintf("%s/ledgers/%d", c.URL, ledgerNum))
```

**Comment by nullstyle** [comment]
Please run `gofmt` on this PR.

**Comment by nullstyle** [comment]
-1.  The strategy we take for representing amounts is something we should consider together.  There are tradeoffs we should consider before we off-handedly break backwards compatibility.

**Comment by bartekn** [comment]
Closing stale PRs.

**Pull request closed without merging**